packages/mjml-core/src/helpers/mediaQueries.js

```diff
@@ -18,7 +18,7 @@ export default function buildMediaQueriesTags(breakpoint, mediaQueries = {}, for
 
   return `
     <style type="text/css">
-      @media only screen and (min-width:${breakpoint}) {
+      @media only screen and (min-width:${breakpoint}), print {
```

Comment:
Maybe we can isolate them inside it's own style to ensure it won't break anything and add an attribute on `mjml` tag to disable it by default and only add them if needed like owa media queriees

Reply:
I like the idea of disabling it by default. I'll try to figure out a way to achieve your suggestion.

Reply:
@iRyusa I applied what you've suggested and it worked perfectly. Like me know if it needs modification and what would the next steps.